public/app/features/trails/ActionTabs/MetricOverviewScene.tsx

```diff
@@ -70,6 +71,13 @@ export class MetricOverviewScene extends SceneObjectBase<MetricOverviewSceneStat
     const { loading: labelsLoading } = variable.useState();
     const labelOptions = getLabelOptions(model, variable).filter((l) => l.value !== ALL_VARIABLE_VALUE);
 
+    // Get unit name from the metric name
+    const metricScene = getMetricSceneFor(model);
+    const metric = metricScene.state.metric;
+    let unit = getUnitFromMetric(metric);
+    if (unit === null) {
+      unit = 'Unknown';
+    }
```

Comment:
```suggestion
    let unit = getUnitFromMetric(metric) ?? 'Unknown';
```

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing

Reply:
oh this is a good suggestion!